pkg/ccl/changefeedccl/sink_kafka_v2.go

```diff
@@ -68,18 +68,19 @@ func newKafkaSinkClientV2(
 	ctx context.Context,
 	clientOpts []kgo.Opt,
 	batchCfg sinkBatchConfig,
-	bootstrapAddrs string,
+	bootstrapAddrsStr string,
```

Comment:
nit: I prefer the old name, I think it's unnecessary to encode the var type in the name here

Reply:
i agree generally that you dont need to encode types in names, but this is how i've always named things that are strings that are about to be parsed. eg 

```
// given line := "hello=42"
parts := strings.Split(line, "=")
key, valStr := parts[0], parts[1]
val, err := strconv.Atoi(numStr)
```

this is probably just a *me* convention...... i'm open to alternatives though.

Reply:
hmm yeah that's fair, although to me the fact that the name is plural (i.e. `...Addrs`) already suggests that it's probably a comma-separated list. maybe you could make the parameter type a `[]string` and do the splitting one level up 😛